presidio-image-redactor/README.MD

```diff
@@ -1,18 +1,101 @@
 # Presidio Image Redactor
 
+***Please notice, this package is still in alpha and not production ready.***
+
 ## Description
 
-The Presidio Image Redactor is a Python based module for detecting and redacting PII text
-entities in images.
+The Presidio Image Redactor is a Python based module for detecting and redacting PII
```

Comment:
Maybe add the flow diagram from the ADR?

Reply:
Done.